sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/implementation/models/CommunicationError.java

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.communication.identity.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+/** The Communication Services error. */
+@Fluent
+public final class CommunicationError {
```

Comment:
Does this need to be public?

Reply:
I did a global search on this and did not see it being used in the APIs, is this public in other langs?

Reply:
This is generated code from swagger. I remove 'public' the code still compiles. But do we want to change the generated code though?

Reply:
Oops sorry, I totally misread Reza's initial comment. Since it is in implementation/models, it will not be exported. So public here really means "internal", which is I think what we want for this.